tests/test_install.py

```diff
@@ -1,15 +1,17 @@
 """Tests related to installing a clang tool."""
-from pathlib import PurePath, Path
 import os
+from pathlib import Path
+from pathlib import PurePath
+
 import pytest
-from clang_tools import install_os, suffix
-from clang_tools.install import (
-    is_installed,
-    clang_tools_binary_url,
-    install_dir_name,
-    create_sym_link,
-    install_tool,
-)
+
+from clang_tools import install_os
+from clang_tools import suffix
+from clang_tools.install import clang_tools_binary_url
+from clang_tools.install import create_sym_link
+from clang_tools.install import install_dir_name
+from clang_tools.install import install_tool
+from clang_tools.install import is_installed
```

Comment:
Personally, I'm not a fan of multiple import statements from the same module, but if this is the convention you want to use then i can adapt. I think it might cause slower execution though. I also noticed the alphabetical ordering here (no objections to that).

Is this on advice from flake8?

Reply:
This is on advice from https://github.com/asottile/reorder_python_imports
I can remove `reorder_python_imports` and revert the format commit, than reformat again.

Reply:
After reviewing what that hook does, I think its more dangerous than it is worth. Messing with imports can be a breaking change and should be managed by the developer. the deprecation of std libs (like `distutils`) is a prime example of how using that could go wrong.

Reply:
See the example from [isort](https://github.com/PyCQA/isort) it seems fit your requirements than reorder_python_imports.

Reply:
Yeah, I guess it would be better to use `isort`.